orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -4818,3 +4818,38 @@ def run_step(model, x):
     _test_helpers.assert_values_are_close(pt_loss, ort_loss)
     _test_helpers.assert_values_are_close(pt_x.grad, ort_x.grad)
     assert ortmodule_module.ONNX_OPSET_VERSION == DEFAULT_OPSET
+
+
+def test_random_states_unchanged_for_ortmodule():
+    import numpy
+    class NeuralNetSlice(torch.nn.Module):
+        def __init__(self):
+            super(NeuralNetSlice, self).__init__()
+            self.dim = 32
+
+        def forward(self, x):
+            # This slice operation will call sympy.Min() when exporting
```

Comment:
Then sympy.Min() internally changed the state, right? Can we make this comment complete, for guys to understand the problem. :)

Reply:
Oh, you'd better add following env in case ORTModule runs failed caused the UT to fallback to PyTorch. 

os.environ['ORTMODULE_FALLBACK_POLICY'] = "FALLBACK_DISABLE"
os.environ['ORTMODULE_FALLBACK_RETRY'] = "False"

Reply:
Found `ORTMODULE_FALLBACK_POLICY` is already taken care of [here](https://github.com/microsoft/onnxruntime/blob/040c0645e2ce600df1284d314976f797fc5f61c0/orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py#L351). Added `ORTMODULE_FALLBACK_RETRY` only.